docker-compose-files/compose.sh

```diff
@@ -229,16 +191,79 @@ echo "GROUP_ID=$(id -g)" >> $topdir/.env
 echo "DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=${DOCKER_INFLUXDB_INIT_ADMIN_TOKEN}" >> $topdir/.env
 echo "DOCKER_INFLUXDB_INIT_PASSWORD=${DOCKER_INFLUXDB_INIT_PASSWORD}" >> $topdir/.env
 
+# init Splink env variables if not set to avoid warnings from docker-compose for other pumps
+if [ -z $SPLUNK_HEC_URL ]; then
+   export SPLUNK_HEC_URL=
+fi
+if [ -z $SPLUNK_HEC_KEY ]; then
+   export SPLUNK_HEC_KEY=
+fi
+if [ -z $SPLUNK_HEC_INDEX ]; then
+   export SPLUNK_HEC_INDEX=
+fi
+
+ # remove dependency on setup influx-test-db
+touch $topdir/docker-compose-files/container-info-influx-pump.txt 
+touch $topdir/docker-compose-files/container-info-grafana.txt
+
 case $1 in
   rm)
     docker-compose --project-directory $topdir -f $scriptdir/docker-compose.yml ${PROFILE_ARG} rm -f
     ;;
 
+  setup)
```

Comment:
little bit problematic to use 'setup' for influx setup. If we have other setup needed in the future, this becomes ambiguous.